### ISSUE TITLE
TypeError: 'bool' object is not callable

### ISSUE THREAD

**Issue by pnh81** [spyder-ide/spyder#11847]
### What steps will reproduce the problem?

During startup, Spyder4 ('mainwindow.py') reports a type error where mod.PLUGIN_CLASS(self) appears to be calling a bool-type object.

### Traceback
```python-traceback
  File "c:\program files\python\lib\site-packages\spyder\app\mainwindow.py", line 985, in setup
    plugin = mod.PLUGIN_CLASS(self)
TypeError: 'bool' object is not callable
```

## Versions

* Spyder version: 4.1.0 
* Python version: 3.8.0
* Qt version: 5.12.6
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.2.2 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.10.1 (OK)
jedi =0.14.1                   :  0.14.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
paramiko >=2.4.0               :  2.7.0 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.7.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.6.7 (OK)
pygments >=2.0                 :  2.5.2 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.2;<0.32.0          :  0.31.2 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.6.0 (OK)
qtconsole >=4.6.0              :  4.6.0 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
sphinx >=0.6.6                 :  2.2.2 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.15 (OK)
matplotlib >=2.0.0             :  3.2.0 (OK)
numpy >=1.7                    :  1.18.2 (OK)
pandas >=0.13.1                :  1.0.2 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by ccordoba12** [comment]
Did you have third-party plugins (like spyder-notebook or spyder-terminal) installed?

**Comment by pnh81** [comment]
Hi Carlos,

no third-party plugins were/are installed. Apart from the packages listed above, the following are also installed:

opencv-python
opencv-contrib-python
imutils
scikit-image
numba

Hope this helps to narrow down the causes for the issue.

Best,
Peter

**Comment by ccordoba12** [comment]
This could be a problem with Python 3.8. @dalthviz, please see if you can reproduce this one.

**Comment by dalthviz** [comment]
Notes:
* Installing using conda in Python 3.8 Spyder doesn't start (blank screen)
* Seems like OP has a conda installation to create the env but uses pip for actually installing the packages (has Qt 5.12.6)
* Maybe the issue is somehow related with pip installations (since #11867 uses pip too)

**Comment by pnh81** [comment]
> Notes:
> 
>     * Installing using conda in Python 3.8 Spyder doesn't start (blank screen)
> 
>     * Seems like OP has a conda installation to create the env but uses pip for actually installing the packages (has Qt 5.12.6)

No usage of Conda whatsoever. Installation was done exclusively with pip. 

> 
>     * Maybe the issue is somehow related with pip installations (since #11867 uses pip too)

This appears to be a possible cause. A fresh install yielded the same problem.

**Comment by ccordoba12** [comment]
@pnh81, thanks for the update.

@dalthviz, please try with pip then.

**Comment by arnyeinstein** [comment]
I reinstalled with pip, but the problem remains. How can I found out which add-ins I have installed, so I could uninstall them?

**Comment by ccordoba12** [comment]
That list is available in `Help > Dependencies` in our 4.1.1 version.

**Comment by pnh81** [comment]
Update: Spyder 4.1.3 fixed the problem for me.

**Comment by ccordoba12** [comment]
Really great to hear @pnh81! I think this was caused by us not supporting Python 3.8 on Windows, which was fixed precisely in 4.1.3.

Therefore, I'm going to close this for now (we haven't seen any other reports similar to this one in a while either).
